src/components/dialog/dialog.svelte

```diff
@@ -42,6 +45,20 @@
     isOpen = false
     onClose?.()
   }
+
+  const setSize = (matches: boolean) => {
```

Comment:
this could be scoped to `onMount`

Reply:
Not opposed, but what's the advantage in this case?

Reply:
Just keeps it defined where its used :smile: